src/cgame/cg_ents.cpp

```diff
@@ -1381,3 +1381,59 @@ void CG_AddPacketEntities()
 		}
 	}
 }
+
+// see also G_Team in src/sgame/sg_ents.cpp
+team_t CG_Team(const centity_t *ent)
+{
+	const entityState_t &es = ent->currentState;
+	switch (es.eType)
+	{
+		case entityType_t::ET_PLAYER:
+		case entityType_t::ET_CORPSE:
+		{
+			clientInfo_t &ci = cgs.clientinfo[es.clientNum];
+			return (team_t) ci.team;
+		}
+		case entityType_t::ET_BUILDABLE:
+			return (team_t) ent->currentState.modelindex2;
+		default:
+			return TEAM_NONE;
```

Comment:
Maybe there should be an assert or something, if it was used on an entity of the wrong type.